src/azure-cli/requirements.py3.windows.txt

```diff
@@ -113,13 +113,15 @@ msal-extensions==1.0.0
 msal==1.18.0b1
 msrest==0.7.1
 msrestazure==0.6.4
+mycli==1.22.2
 oauthlib==3.0.1
 packaging==21.3
 paramiko==2.10.1
 pbr==5.3.1
 pkginfo==1.8.2
 portalocker==2.3.2
 psutil==5.9.0
+psycopg2==2.9.3
```

Comment:
Why is it required for windows only?

Reply:
psycopg2 has no pre-compiled binary version on Macos and some linux distribution. It needs to compile from source code and needs some extra library installed. In order not to impact too many CLI users, we will install packages dynamically when users run the specific command.

Reply:
`psycopg2` currently cannot be installed on Python 3.11 (https://github.com/psycopg/psycopg2/issues/1514), which blocks Azure CLI's Python 3.11 support (https://github.com/Azure/azure-cli/issues/24494).